event/event.go

```diff
@@ -17,20 +17,12 @@ type Event struct {
 
 // Create creates an event.
```

Comment:
More docs about an create event - it creates based on event definition in service and checks if all data are valid/exist.

Reply:
done